cftime/src/macros.rs

```diff
@@ -0,0 +1,48 @@
+#[macro_export]
```

Comment:
Remove macro_export, put annotation `#[macro_use]` above `mod macros` which has to be the top import in `lib.rs`. After removing all `use` statements of the macros the macro will not be exported outside this crate